findimports.py

```diff
@@ -625,7 +628,7 @@ def listModules(self):
         """Return an alphabetical list of all modules."""
         modules = list(self.modules.items())
         modules.sort()
-        return [module for name, module in modules]
+        return [module for _, module in modules]
```

Comment:
I prefer the expressiveness of `name, module` to remind the reader about the shape of the data structure, even if `name` is unused here.

Is there a linter that shows a warning about an unused name here?  If yes, can we silence it by using `_name`?  If not, I'd rather revert this bit.

Reply:
No problem! I understand you point of view. Let's revert it.